src/web/components/ParticipantRequests/ParticipantRequestItem.tsx

```diff
@@ -49,9 +49,7 @@ export function ParticipantRequestItem({
         <td>
           <div className='participant-request-name-cell'>
             <img src={logo} alt={participant.name} className='participant-request-logo' />
-            <label htmlFor={`checkbox-${participant.id}`} className='checkbox-label'>
-              {participant.name}
-            </label>
+            {participant.name}
```

Comment:
The browser was throwing an HTML validation error because the label didn't refer to anything.  It looks like the html may have changed at some point in the past.